src/numbers.js

```diff
@@ -1,45 +1,59 @@
 function add (a, b) {
   // your code here
+  return a + b;
 }
 
 function subtract (a, b) {
   // your code here
+  return a - b;
 }
 
 function multiply (a, b) {
   // your code here
+  return a * b;
 }
 
 function divide (a, b) {
   // your code here
+  return a / b;
 }
 
 function power (a, b) {
   // your code here
+  return a ** b;
 }
 
 function round (a) {
   // your code here
+  return Math.round(a);
 }
 
 function roundUp (a) {
   // your code here
+  return Math.ceil(a);
 }
 
 function roundDown (a) {
   // your code here
+  return Math.floor(a);
 }
 
 function absolute (a) {
   // your code here
+  return Math.abs(a);
 }
 
 function quotient (a, b) {
   // your code here
+  if(a < 0 || b < 0) {
+    return Math.ceil(a / b);
+  }
+  return Math.floor(a / b);
```

Comment:
Looking at what a quotient is as defined in the test:
```
  // N.B. quotient of two numbers is the integer given by dividing
  // the first by the second, without the remainder
  // 18 divided by 7 is 2 remainder 4 (or 2.571...)
  // so the quotient of 18 and 7 is 2
```
What we are looking to do is less about rounding numbers up or down, but simply removing anything after the decimal point. There is a built in function in JS to do just that (`Math.trunc()`) which will simplify this a bit:

```suggestion
  return Math.trunc(a / b);
```